homeassistant/components/jellyfin/media_player.py

```diff
@@ -0,0 +1,295 @@
+"""Support for the Jellyfin media player."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.media_player import (
+    MediaPlayerEntity,
+    MediaPlayerEntityDescription,
+    MediaPlayerEntityFeature,
+    MediaPlayerState,
+    MediaType,
+)
+from homeassistant.components.media_player.browse_media import BrowseMedia
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.util.dt import parse_datetime
+
+from .browse_media import build_item_response, build_root_response
+from .client_wrapper import get_artwork_url
+from .const import CONTENT_TYPE_MAP, DOMAIN
+from .coordinator import JellyfinDataUpdateCoordinator
+from .entity import JellyfinEntity
+from .models import JellyfinData
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Jellyfin media_player from a config entry."""
+    jellyfin_data: JellyfinData = hass.data[DOMAIN][entry.entry_id]
+    coordinator = jellyfin_data.coordinators["sessions"]
+
+    async_add_entities(
+        (
+            JellyfinMediaPlayer(coordinator, session_id, session_data)
+            for session_id, session_data in coordinator.data.items()
+            if session_data["DeviceId"] != jellyfin_data.client_device_id
+            and session_data["Client"] != "Home Assistant"
```

Comment:
Is `"Home Assistant"` the `USER_APP_NAME`? I'd use that constant in that case.

Reply:
https://github.com/home-assistant/core/pull/80987